db/scripts/schema.sql

```diff
@@ -0,0 +1,10 @@
+DROP SCHEMA IF EXISTS `rso`;
+CREATE SCHEMA IF NOT EXISTS `rso`;
+USE rso;
+
+CREATE TABLE IF NOT EXISTS client
+(
+    id    BIGINT(20)  NOT NULL AUTO_INCREMENT,

```

Comment:
With JPA it is better to use sequences for IDs

Reply:
It's only example how to load schema and data to container. Data scripts will be replaced for sure.